### PR TITLE
Add nunit test VS test runner setting to project files

### PR DESCRIPTION
**Pull request by bording**
Some of the test projects had this already, so this lets all of them work with the VS Test Explorer.

### CODE REVIEW

File: src/NServiceBus.sln [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+VisualStudioVersion = 14.0.25123.0
```
Comment (danielmarbach) [inline src/NServiceBus.sln]: Is this change necessary?

Comment (bording) [inline src/NServiceBus.sln]: No, but given that VS will want to update that every time someone does something that touches the solution file anyway, I didn't see the harm in letting it update it.

I can remove it if you'd prefer.

Comment (danielmarbach) [inline src/NServiceBus.sln]: just curious
